js/wp-seo-metabox.js

```diff
@@ -219,16 +333,16 @@ function yst_updateTitle(force) {
 	}
 
 	title = yst_replaceVariables(title, function (title) {
+		title = sanitize_title(title);
+
+		jQuery('#wpseosnippet_title').html(title);
+
 		// do the placeholder
 		var placeholder_title = divHtml.html(title).text();
 		titleElm.attr('placeholder', placeholder_title);
 
-		title = sanitize_title( title );
-
-		jQuery('#wpseosnippet_title').html(title);
-
 		var e = document.getElementById('wpseosnippet_title');
```

Comment:
I wouldn't use <code>e</code> as variable name, since <code>e</code> is usually used for events.